contracts/consensus/Consensus.sol

```diff
@@ -662,7 +740,8 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
             "Core lifetime status must be genesis or active."
         );
 
-        CoreI(_core).logout(msg.sender);
+        _core.logout(msg.sender);
+
         reputation.deregister(msg.sender);
```

Comment:
@deepesh-kn @jayeshbairagi can we make a ticket to make sure that (co)consensus *first* calls reputation, then the core? not the other way around, thx